.github/workflows/juno-test.yml

```diff
@@ -42,8 +42,13 @@ jobs:
       - name: Install dependencies (macOS)
         if: runner.os == 'macOS'
         run: brew install jemalloc
-        
-      - name: Tests
+
+      - name: Tests (Coverage)
+        if: matrix.os == 'ubuntu-latest'
+        run: make test-cover
+
+      - name: Tests (No Coverage)
+        if: matrix.os != 'ubuntu-latest'
```

Comment:
also question regarding target definition:
```
test: clean-testcache rustdeps ## tests
	go test $(GO_TAGS) ./...

test-cover: rustdeps ## tests with coverage
        mkdir -p coverage
        go test $(GO_TAGS) -coverpkg=./... -coverprofile=coverage/coverage.out -covermode=atomic ./...
        go tool cover -html=coverage/coverage.out -o coverage/coverage.html
```

Reply:
seems like it make sense, let me add it